util.py

```diff
@@ -301,3 +302,43 @@ def instanciate_delayed(o):
         for k in o:
             o[k] = instanciate_delayed(o[k])
     return o
+
+
+def get_executable_path(
+    path: tk.Path, gs_member_name: str, default_exec_path: Optional[tk.Path] = None
+) -> tk.Path:
+    if path is not None:
+        if isinstance(path, tk.Path):
+            return path
+        elif isinstance(path, str):
```

Comment:
We could issue a warning here that it is not recommended/deprecated to use strings.

Reply:
yes.. and also a warning if the `gs_member_name` is used..?